src/data/content/design-system/come-contribuire.yaml

```diff
@@ -85,6 +85,8 @@ components:
   filterCards:
     id: esplora
     title: Esplora le schede
+    designSystem: true
```

Comment:
We need this field to apply alignment and spacing classes to the `filterCards` and `sectionsEditorial` components when used in the design system section, the only section with a left navigation sidebar. 
Maybe we can do this during build knowing where you are... but it will be a first for us. 

ps. I know that currently we only use the `filterCards` component in the design system section, but to make it usable elsewhere we need the field for it too.

Reply:
Maybe we can rename it to: `designSystemMode` or `designSystemPage` if you prefer.

Reply:
maybe the attribute needs to explain the function better, regardless of the page, sort of `reduceSpace` or `increaseSpacing` (depending on what it really does). 

Today, we could need it on DS pages, tomorrow...who knows :)

Reply:
@zetareticoli exactly. A `reducedSpace` filter card is something intuitive and self-documenting. Also, spacing can be a property of that card object. Where it is used not so much.

We have a `noSpace` property as well, maybe we can unify all the spacing notions into one.